### DISCUSSION TITLE
Issues with FormR: Access codes do not work; Login not possible

### DISCUSSION THREAD

**Discussion by AlChTr** [rubenarslan/formr.org#565] [General]
Hi all,
Currently we are facing two problems:

1. the individual access codes do not work: 
publicness is set on:“You and people who have an access code”. However, users with a personal access code (link with individual session ID), cannot return to their respective position in the run. Instead, they start from the beginning as new users.

2. We are currently unable to log in to our FormR accounts: When trying to log in, the login page just reloads every time

Best regards,
Alex

**Comment by jkelgs** [comment]
Dear all,

It looks like the issue is still ongoing, and our participants can't fill out surveys. Does anyone have an idea when it might be resolved?

Thanks!

**Comment by rubenarslan** [comment]
It seems @cyriltata has been able to resolve this issue. We encourage self hosting, as the formr.org instance is experiencing too much load. Write me, if you're considering this.
